proxy/http/HttpTransact.cc

```diff
@@ -5807,6 +5783,34 @@ HttpTransact::initialize_state_variables_from_request(State *s, HTTPHdr *obsolet
   s->dns_info.lookup_name = s->server_info.name;
 }
 
+void
```

Comment:
While we're at it, would this not be better / nicer with a switch statement ?

Reply:
I tried, but values like `HTTP_WKSIDX_GET` is not compile-time constants so I left it the way it was.